src/components/AdminDashboard/FormResponseIncidents.js

```diff
@@ -1,16 +1,10 @@
 import React, { useState, useEffect } from 'react';
```

Comment:
Not sure this components is being used.  Looks like a shell of a component that somehow got merged previously.